VIMVideoPlayer/VIMVideoPlayer.m

```diff
@@ -95,8 +95,13 @@ - (instancetype)init
 - (void)setupPlayer
 {
     self.player = [[AVPlayer alloc] init];
-    self.player.muted = NO;
-    self.player.allowsExternalPlayback = YES;
```

Comment:
Do we absolutely need to make these changes to VIMVideoPlayer right now? If not, I'd prefer to defer until after v6.0 shipment